apps/src/templates/progress/ProgressLessonTeacherInfo.story.jsx

```diff
@@ -156,7 +169,7 @@ export default storybook => {
             <Provider store={store}>
               <div style={style}>
                 <ProgressLessonTeacherInfo
-                  lesson={lessons(state.progress)[1]}
+                  lesson={lessons(state.progress)[3]}
```

Comment:
Interestingly, this story wasn't actually doing what it said it was doing in the description (as it was using a stage that did have a lesson plan).